core/pages/Category.js

```diff
@@ -29,7 +29,7 @@ export default {
     ...mapGetters('category', ['getCurrentCategory', 'getCurrentCategoryProductQuery', 'getAllCategoryFilters', 'getCategoryBreadcrumbs', 'getCurrentCategoryPath']),
     ...mapGetters('tax', ['getIsUserGroupedTaxActive']),
     products () {
-      return this.$store.getters['product/list']
+      return this.$store.getters['product/findProducts']
```

Comment:
please keep previous one. Category.js is not used anymore and `this.$store.getters['product/findProducts']` doesn't exist

Reply:
Thanks @gibkigonzo for review. 

Done

Reply:
thanks :)